src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String SPACE_SPLITTER = " ";
+    private static final int DATE_COLUMN = 0;
+    private static final int EMPLOYEE_COLUMN = 1;
+    private static final int HOUR_COLUMN = 2;
+    private static final int SALARY_COLUMN = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        String[] dataBase;
+        int salary;
+
+        LocalDate fromAsDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate toAsDate = LocalDate.parse(dateTo, formatter);
+
+        result.append(String.format("Report for period %s - %s", dateFrom, dateTo));
```

Comment:
```suggestion
        int salary;
        LocalDate fromAsDate = LocalDate.parse(dateFrom, formatter);
        LocalDate toAsDate = LocalDate.parse(dateTo, formatter);
        result.append(String.format("Report for period %s - %s", dateFrom, dateTo));
```

lets remove redundant empty lines